ang/crmMailing/Templates.js

```diff
@@ -98,7 +98,7 @@
                       results.text = ts('Message Templates');
                     }
 
-                    more = data.more_results;
+                    var more = data.more_results;
 
                     if (more && !data.more_results) {
```

Comment:
Maybe out-of-scope for this PR, but I just did a double take at the logic of these 2 lines. What the :hankey:???
:laughing:

Reply:
This fixes it: https://github.com/civicrm/civicrm-core/pull/23188